src/tests/integration/MetaMorpho.t.sol

```diff
@@ -0,0 +1,210 @@
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+pragma solidity ^0.8.14;
+
+import "forge-std/Test.sol";
+import "./IntegrationBase.t.sol";
+import "morpho-blue/src/interfaces/IMorpho.sol";
+import "morpho-blue/src/libraries/MarketParamsLib.sol";
+import "morpho-blue/src/libraries/periphery/MorphoBalancesLib.sol";
+import "metamorpho/libraries/PendingLib.sol";
+import "forge-std/interfaces/IERC4626.sol";
+
+// Versioning issues with import, so it's inline
+interface IMetaMorpho is IERC4626 {
+    function owner() external view returns (address);
+    function timelock() external view returns (uint256);
+    function withdrawQueue(uint256) external view returns (Id);
+    function withdrawQueueLength() external view returns (uint256);
+    function config(Id) external view returns (MarketConfig memory);
+    function setSupplyQueue(Id[] calldata newSupplyQueue) external;
+    function submitCap(MarketParams memory marketParams, uint256 newSupplyCap) external;
+    function acceptCap(MarketParams memory marketParams) external;
+}
+
+contract MetaMorphoTest is IntegrationBaseTest {
+    using MorphoBalancesLib for IMorpho;
+    using MarketParamsLib for MarketParams;
+
+    IMetaMorpho constant spDai = IMetaMorpho(0x73e65DBD630f90604062f6E02fAb9138e713edD9);
+    address constant sUsde = 0x9D39A5DE30e57443BfF2A8307A4256c8797A3497;
+    IMorpho constant morpho = IMorpho(0xBBBBBbbBBb9cC5e90e3b3Af64bdAF62C37EEFFCb);
+    D3MOperatorPlan plan;
+    D3M4626TypePool pool;
+
+    // sUSDe/USDC (91.5%).
+    MarketParams public marketParams = MarketParams({
+        loanToken: 0x6B175474E89094C44Da98b954EedeAC495271d0F,
+        collateralToken: 0x9D39A5DE30e57443BfF2A8307A4256c8797A3497,
+        oracle: 0x5D916980D5Ae1737a8330Bf24dF812b2911Aae25,
+        irm: 0x870aC11D48B15DB9a138Cf899d20F13F79Ba00BC,
+        lltv: 915000000000000000
+    });
+    // sUSDe/USDC (94.5%).
+    MarketParams public marketParamsHighLltv = MarketParams({
+        loanToken: 0x6B175474E89094C44Da98b954EedeAC495271d0F,
+        collateralToken: 0x9D39A5DE30e57443BfF2A8307A4256c8797A3497,
+        oracle: 0x5D916980D5Ae1737a8330Bf24dF812b2911Aae25,
+        irm: 0x870aC11D48B15DB9a138Cf899d20F13F79Ba00BC,
+        lltv: 945000000000000000
+    });
+
+    address operator = makeAddr("operator");
+    uint256 constant startingAmount = 5_000_000 * WAD;
+
+    function setUp() public {
+        baseInit();
+
+        vm.createSelectFork(vm.envString("ETH_RPC_URL"), 19456934);
+
+        // Need to increase to 3
+        roundingTolerance = 3;
```

Comment:
All the tests are passing for me setting this to `1`.

Reply:
Must have been resolved with the other changes. Reverted.